javascript/new-src/Gruntfile.js

```diff
@@ -18,7 +18,24 @@ module.exports = function(grunt) {
       build: {
         files: [{
           dest: '../../olad/www/new/js/app.min.js',
-          src: 'src/js/app.js'
+          src: [
```

Comment:
Can you not define this array outside of the function to avoid duplication if/when you need to add more files to it?